### ISSUE TITLE
Your input tensor xs has a size of 174182400, which can't be trans to [1, 4, -1, 921600] ([B, K, -1, L]). The xs gained from `xs = torch.stack([x1, x2, x3, x4],dim=1)` is supposed to be divisible by 4. Did you change anything?

### ISSUE THREAD

**Issue by Yolo1-gguo** [aSleepyTree/FreqMamba#4]
Your input tensor xs has a size of 174182400, which can't be trans to [1, 4, -1, 921600] ([B, K, -1, L]). The xs gained from `xs = torch.stack([x1, x2, x3, x4],dim=1)` is supposed to be divisible by 4. Did you change anything?

_Originally posted by @aSleepyTree in https://github.com/aSleepyTree/FreqMamba/issues/3#issuecomment-2421369630_

**Comment by Yolo1-gguo** [comment]
Hello, I used my own dataset and used your freqmamba _ arch module, and nothing has been changed to the things inside this module.

**Comment by aSleepyTree** [comment]
I don't understand. The second dimension of xs is certain to be 4, and maybe you should check the intermediate variables.
